service/src/main/java/org/broadinstitute/listener/relay/Utils.java

```diff
@@ -69,11 +74,11 @@ public static boolean isValidOrigin(String origin, CorsSupportProperties corsSup
     String finalStringToCheck = stringToCheck;
```

Comment:
Commenting here because GH won't let me comment on line 66 (idk): can we demote these to debug logs once validated? They're super spammy.